people/people_service.go

```diff
@@ -62,26 +60,17 @@ func (s service) Read(uuid string) (interface{}, bool, error) {
 
 	result := results[0]
 
-	p := person{
-		UUID:       result.UUID,
-		Name:       result.Name,
-		BirthYear:  result.BirthYear,
-		Salutation: result.Salutation,
-		Aliases:    result.Aliases,
-	}
-
-	if result.FactsetIdentifier != "" {
-		p.Identifiers = append(p.Identifiers, identifier{fsAuthority, result.FactsetIdentifier})
+	if len(result.Identifiers) == 1 && (result.Identifiers[0].IdentifierValue == "") {
+		result.Identifiers = make([]identifier, 0, 0)
```

Comment:
"make([]identifier, 0, 0)" can just become "[]identifier{}" which would be more idiomatic, and possible avoid memory allocation